languages/it.ini

```
@@ -35,3 +35,5 @@ help = Come usare YAPI
 
 [SCRIPT-RUNNER]
 0_package_not_found_error = Pacchetto non trovato. Riprova.
+1_package_intalled = Pacchetto installato correttamente
+2_package_error = Qualcosa è andato storto durante l'installazione
```

```suggestion
2_package_error = Qualcosa è andato storto durante l\'installazione
```
Insert escape sequence for apostrophe.